modules/primer-marketing-buttons/README.md

```diff
@@ -0,0 +1,105 @@
+# Primer Marketing CSS Buttons
+
+[![npm version](http://img.shields.io/npm/v/primer-marketing-buttons.svg)](https://www.npmjs.org/package/primer-marketing-buttons)
+[![Build Status](https://travis-ci.org/primer/primer-css.svg?branch=master)](https://travis-ci.org/primer/primer-css)
+
+> Buttons are used for actions, like in forms, while textual hyperlinks are used for destinations, or moving from one page to another.
+
+This repository is a module of the full [primer-css][primer] repository.
+
+## Documentation
+
+<!-- %docs
+title: Marketing Buttons
+status: New Release
+-->
+
+The buttons on our marketing pages are based on Primer's core with the addition of more color options, color outline buttons, a transparent button, and larger button sizes.
+
+
+## Color
+
+The `btn` has been extended with the marketing color palette:
+
+```html
+
+<button class="btn btn-orange" type="button">Button button</button>
+<a class="btn btn-purple" href="#url" role="button">Link button</a>
+<button class="btn btn-blurple" type="button">Button button</button>
+
+```
+
+## Outline buttons
+
+The `btn-outline` has also been extended with the marketing color palette:
+
+```html
+
+<button class="btn btn-outline-orange" type="button">Button button</button>
+<a class="btn btn-outline-purple" href="#url" role="button">Link button</a>
+<button class="btn btn-outline-green" type="button">Button button</button>
+<div class="bg-gray-dark p-4 mt-4">
+  <button class="btn btn-transparent" type="button">
+    Button button
+    <span class="Counter">12</span>
+  </button>
+</div>
+
+```
+
+## Additional sizing
+There are cases where you might want to increase the size of a button, for example when putting a main CTA inside of a jumbotron or major page callout. The `btn-large` class does the following to make a button more prominent:
```

Comment:
I think the intro paragraph can be more concise, and I don't think we need to call out the description in a numbered list or get into exact values, suggest changing to something like the following:

> Use `btn-large` to increase the padding and border radius of a button. This is useful for prominent calls to action, and is often used in combination with [jumbtron](??) styles. 

> [Type scale utilities](typography/#type-scale-utilities) can be used to alter the font-size if needed. Padding is applied in em's so that it scales proportionally with the font-size.

**Side note:** As someone less familiar with marketing styles, I had no idea what a `jumbotron` might be. I looked it up and this looks like it could be called `hero` which I think is more meaningful to people. We try to stick with class names that are obvious, transparent, and human-readable—could we change this to something more meaningful in future? Related, `minitron` and `supertron` could just me `small` and `large` :woman_shrugging:. I get this is probably just a bit of humor, but it's un-helpful for people learning to use Primer. cc @sophshep

Reply:
I've changed the language in this section of the docs to "This is useful for prominent calls to action in hero sections."

Related, this section of the docs pertaining to `btn-large` is also moved into primer-buttons' readme.md as `btn-large`has been moved back to primer-buttons as well, where it originally was just without documentation.